avocado_vt/loader.py

```diff
@@ -116,6 +131,11 @@ def get_extra_listing(self):
             args.vt_config = None
             args.vt_guest_os = None
             guest_listing(args)
+        if self.args.vt_list_archs:
```

Comment:
OK, looks good to optionally add information, that is, optionally show guests and/or archs.